img/s/studentaid.gov.svg

```diff
@@ -0,0 +1,554 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   version="1.1"
+   id="svg160"
+   width="180"
+   height="180"
+   viewBox="0 0 180 180"
+   sodipodi:docname="studentaid.gov.svg"
+   inkscape:version="1.2.1 (9c6d41e410, 2022-07-14)"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:xlink="http://www.w3.org/1999/xlink"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:svg="http://www.w3.org/2000/svg">
+  <defs
+     id="defs164" />
+  <sodipodi:namedview
+     id="namedview162"
+     pagecolor="#ffffff"
+     bordercolor="#000000"
+     borderopacity="0.25"
+     inkscape:showpageshadow="2"
+     inkscape:pageopacity="0.0"
+     inkscape:pagecheckerboard="0"
+     inkscape:deskcolor="#d1d1d1"
+     showgrid="false"
+     inkscape:zoom="4.65"
+     inkscape:cx="90.107527"
+     inkscape:cy="90.107527"
+     inkscape:window-width="1920"
+     inkscape:window-height="1017"
+     inkscape:window-x="1912"
+     inkscape:window-y="-8"
+     inkscape:window-maximized="1"
+     inkscape:current-layer="g166" />
+  <g
+     inkscape:groupmode="layer"
+     inkscape:label="Image"
+     id="g166">
+    <image
+       width="180"
+       height="180"
+       preserveAspectRatio="none"
+       style="image-rendering:optimizeQuality"
+       xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAALQAAAC0CAYAAAA9zQYyAAAABGdBTUEAALGPC/xhBQAAADhlWElm
```

Comment:
This SVG is just an embedded PNG file. I suggest you either rename the previous SVG (`studentaid.ed.gov.svg`) or find another SVG that's an actual SVG.
You can use [SVGOMG](https://jakearchibald.github.io/svgomg/) with the setting __Remove raster images__ enabled to see if the SVG-file is a PNG-file in disguise.

Reply:
I found this image https://myfedloan.org/images/fsa/badge-light-right.svg and copied the icon from it. The S seems a bit odd but it's an official logo :shrug:
```SVG/XML
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 22.8 22.8"><defs><linearGradient id="a" x1="0" x2="22.8" y1="11.4" y2="11.4" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#12303b"/><stop offset="1" stop-color="#345065"/></linearGradient></defs><circle cx="11.4" cy="11.4" r="11.4" fill="url(#a)"/><path fill="#fff" d="M4.5195 5.6797V13.98h1.5v-3.8h1.5V9.1797h-1.5V6.7793h1.9004V5.6797H4.5195zm6.2015 0c-.7 0-1.2019.1-1.6019.5-.4.3-.5996.8-.5996 1.5 0 .4.1008.7996.3008 1.0996.2.3.3992.6008.6992.8008l1.0005.8999c.3.2.601.5.701.7.1.2.199.5.199.9 0 .3-.099.5-.199.6-.1.1-.3.201-.5.201s-.4-.101-.5-.201l-.3011-.6c0-.2-.0996-.501-.0996-.801l-1.3008.301c0 .6.1008 1 .3008 1.4.2.3.4988.599.7988.799.3.2.7009.301 1.2009.301.4 0 .8-.101 1.1-.301.3-.2.499-.399.699-.699.2-.4.301-.7.301-1.1 0-.6-.2-1-.4-1.4-.3-.4-.601-.6999-.901-.9999l-1-.9004c-.2-.2-.4-.3996-.5-.5996-.1-.2-.1991-.3992-.1991-.6992 0-.2.0991-.4.1991-.5.1-.1.3-.2012.5-.2012s.4.1012.5.2012c.1.1.201.3.201.5.1.2.1.3992.1.6992l1.301-.1992c0-.2-.002-.5008-.102-.8008 0-.2-.099-.4992-.299-.6992-.1-.2-.299-.4-.599-.5-.3-.1-.6-.2012-1-.2012zm4.299 0-1.7 8.3003h1.6l.301-1.9h1.398l.301 1.9h1.4l-1.701-8.3003H15.02zm.701 1.9004.599 3.4999h-1.099l.5-3.4999z"/><path fill="#43b02a" d="M4.52 16.08h13.8v1.1H4.52Z"/></svg>
```